module/solutions/helper.js

```diff
@@ -983,8 +983,7 @@ module.exports = class SolutionsHelper {
 				// }
 
 				filterQuery.status = CONSTANTS.common.ACTIVE_STATUS
-				filterQuery.type = type
-
+				filterQuery.type = data.type
```

Comment:
@MallanagoudaB what will happen in some places with this function call type is passed as param, it will completely get ignored right. So better to pass data.type as a second arguement where the queryBasedOnRoleAndLocation function is called only with single param